bndtools.core/src/bndtools/editor/BndSourceEditorPage.java

```diff
@@ -189,4 +194,19 @@ private IDocument getDocument() {
     public Image getTitleImage() {
         return icon;
     }
+
+    @Override
+    protected void createActions() {
```

Comment:
Could you send this toggle comment in a separate pull request?  this feature we could merge right away.